docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -152,10 +152,10 @@ $(LOCALBIN):
 
```

Comment:
The right emoji is `:sparkles:` since it change the scaffold provided for the users
🌱  <- Usually we use for recipes or changes that are not user face

More info: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md

So, I hope that you do not mind but I changed the title emoji